packages/vitest/src/create/browser/creator.ts

```diff
@@ -474,19 +440,24 @@ export async function create(): Promise<void> {
   log()
 
   if (rootConfig) {
-    let browserWorkspaceFile = resolve(dirname(rootConfig), `vitest.workspace.${lang}`)
-    if (existsSync(browserWorkspaceFile)) {
-      log(c.yellow('⚠'), c.yellow('A workspace file already exists. Creating a new one for the browser tests - you can merge them manually if needed.'))
-      browserWorkspaceFile = resolve(process.cwd(), `vitest.workspace.browser.${lang}`)
-    }
-    scriptCommand = `vitest --workspace=${relative(process.cwd(), browserWorkspaceFile)}`
-    await generateWorkspaceFile({
-      configPath: browserWorkspaceFile,
-      rootConfig,
+    const configPath = resolve(dirname(rootConfig), `vitest.browser.config.${lang}`)
+    scriptCommand = `vitest --config=${relative(process.cwd(), configPath)}`
+    await generateFrameworkConfigFile({
+      configPath,
+      framework,
+      frameworkPlugin,
       provider,
       browsers,
     })
-    log(c.green('✔'), 'Created a workspace file for browser tests:', c.bold(relative(process.cwd(), browserWorkspaceFile)))
+    log(
+      c.green('✔'),
+      'Created a new config file for browser tests:',
+      c.bold(relative(process.cwd(), configPath)),
+      // TODO: Can we modify the config ourselves?
```

Comment:
For coverage we do something similar with `magicast` when `thresholds.autoUpdate` is set. 🤔